internal/controller/stas/ssa_client.go

```diff
@@ -0,0 +1,144 @@
+package stas
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/sets"
+	metav1ac "k8s.io/client-go/applyconfigurations/meta/v1"
+	"k8s.io/client-go/util/csaupgrade"
+	"k8s.io/utils/ptr"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/apiutil"
+
+	ctrlerrors "github.com/statnett/image-scanner-operator/internal/errors"
+)
+
+const (
+	fieldOwner = client.FieldOwner("image-scanner-operator")
```

Comment:
Don't already have a field manager? We should check the manager name currently used. The same manager must be used, or SSA won't work.